tests/jballerina-unit-test/src/test/resources/test-src/statements/variabledef/globalcycle/src/viaservice/file01.bal

```diff
@@ -22,19 +22,19 @@ string b7a = "b7a";
 int port = o.p;
 
 Obj o = new();
+string str = "";
+int p = port;
 
 service hello on lis {
```

Comment:
Do we need this service at all for this test?

Reply:
Not sure. The `__init()` of the service contains an assignment using global vars. Shall we just leave it as is at least for now? The checks passed previously

Reply:
Please refer https://github.com/ballerina-platform/ballerina-spec/issues/289